frontend/drivers/platform_emscripten.c

```diff
@@ -552,6 +556,8 @@ static void frontend_emscripten_get_env(int *argc, char *argv[],
    char bundle_path[PATH_MAX];
    const char *home = getenv("HOME");
 
+   path_set(RARCH_PATH_CORE, getenv("LIBRARY_PATH"));
```

Comment:
If the calling context doesn't define this environment variable, will this still behave the same as before?